doc/source/whatsnew/v0.20.0.rst

```diff
@@ -396,7 +396,7 @@ IntervalIndex
 
 pandas has gained an ``IntervalIndex`` with its own dtype, ``interval`` as well as the ``Interval`` scalar type. These allow first-class support for interval
 notation, specifically as a return type for the categories in :func:`cut` and :func:`qcut`. The ``IntervalIndex`` allows some unique indexing, see the
-:ref:`docs <indexing.intervallindex>`. (:issue:`7640`, :issue:`8625`)
+:ref:`docs <api.intervalindex>`. (:issue:`7640`, :issue:`8625`)
```

Comment:
```suggestion
:ref:`docs <advanced.intervalindex>`. (:issue:`7640`, :issue:`8625`)
```

I think the original intent was to refer to the user guide section, which is moved to "advanced.rst" (we actually shouldn't have renamed that label in the first place ..)